handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -152,6 +154,15 @@ public SslContextBuilder sslProvider(SslProvider provider) {
     }
 
     /**
+     * The SSLContext {@link Provider} to use. {@code null} uses the default one. This is only
+     * used with {@link SslProvider#JDK}.
+     */
+    public SslContextBuilder sslContextProvider(Provider sslContextProvider) {
+        this.sslContextProvider = sslContextProvider;
```

Comment:
should we validate the "only with JDK SslProvider" condition? Not sure where the right spot for this is (SslContext, here + sslProvider method).